internal/cmd/tests/cmd_run_test.go

```diff
@@ -1952,9 +1952,9 @@ func TestUIRenderWebDashboard(t *testing.T) {
 		active    bool
 		expRender string
 	}{
-		{expRender: "web dashboard:"},
-		{env: "false", expRender: "web dashboard:"},
-		{env: "true", active: true, expRender: "web dashboard: http://127.0.0.1:"},
+		{active: true, expRender: "web dashboard: http://127.0.0.1:"},
```

Comment:
> To support environments where a web server is not desired (e.g., CI/CD),
the logic has been inverted from opt-in to opt-out. The dashboard can
now be explicitly disabled.

What will happen in CI/CD pipelines? Could this change break those?